frontend/apps/reader/modules/readerfont.lua

```diff
@@ -66,7 +67,8 @@ function ReaderFont:init()
                 -- defaults are hardcoded in credocument.lua
                 local default_font = G_reader_settings:readSetting("cre_font") or self.ui.document.default_font
                 local fallback_font = G_reader_settings:readSetting("fallback_font") or self.ui.document.fallback_fonts[1]
-                local text = v
+                local text = FontList:getLocalizedFontName(cre.getFontFaceFilenameAndFaceIndex(v)) or v
```

Comment:
Might do the `cre.getFontFaceFilenameAndFaceIndex(v)` at the start of the for loop, as it's now done for every font, and it might (the default) also be done in the font_func.

Reply:
About `FontList:getLocalizedFontName()` - I wonder if we need an option to _not_ get/show the localized one (in case it is crap and people were fine with the regular names). We might add that later if needed.

Reply:
The field is surprisingly accurate, if I'd wager for user requested option, it would be an option to display *filenames* - as those quite often don't match the "fancy names" of ours now, so its difficult to discern which font is what on storage.